kvbc/include/KVBCInterfaces.h

```diff
@@ -121,7 +121,6 @@ class ICommandsHandler : public bftEngine::IRequestsHandler {
   void execute(ExecutionRequestsQueue& requestList,
                const std::string& batchCid,
                concordUtils::SpanWrapper& parent_span) override = 0;
-  virtual void setControlStateManager(std::shared_ptr<bftEngine::ControlStateManager> controlStateManager) = 0;
```

Comment:
Notice that this breaks API in concord also

Reply:
Yes, we'll do it during bft version bump